.github/workflows/leaves_pr.yaml

```diff
@@ -5,11 +5,29 @@ jobs:
     runs-on: ubuntu-latest
     name: Validate pending leaves
     steps:
+
     - uses: actions/checkout@v2
       with:
          fetch-depth: 0
+
     - name: Leaf validator step
       id: leaf_validator
       uses: google/trillian-examples/serverless/deploy/github/leaf_validator@master
       with:
         log_dir: './log'
+
+    - name: Install golang
```

Comment:
Would it make sense to implement this as its own action? (e.g. compare with https://github.com/google/trillian-examples/tree/master/serverless/deploy/github/sequence_and_integrate)

Reply:
Ok, will do in a separate PR. Thanks